cmd/compute-domain-controller/resourceclaimtemplate.go

```diff
@@ -127,7 +135,7 @@ func (m *BaseResourceClaimTemplateManager) Start(ctx context.Context) (rerr erro
 	m.waitGroup.Add(1)
 	go func() {
 		defer m.waitGroup.Done()
-		m.factory.Start(ctx.Done())
+		m.informer.Run(ctx.Done())
```

Comment:
OK, I trust that this is just interface mangling and no change in behavior.

Reply:
Yeah, since we don't have a factory anymore, we just need to start the informer directly instead of starting the factory.